operator/Makefile

```diff
@@ -241,8 +241,10 @@ manifests: parent-proto-generate controller-gen ## Generate WebhookConfiguration
 .PHONY: generate
 generate: parent-proto-generate controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="../pkg/complianceoperator/..."
```

Comment:
I don't think it's a good idea to couple `/operator` and `/pkg/complianceoperator`. As far as I can tell they are unrelated (apart from the name and similar tooling).
This target should probably go in the top-level `Makefile` instead.

Reply:
Fixed